build/container/Makefile

```diff
@@ -258,7 +258,7 @@ publish/multi-arch-base-manifest :
     	$(DOCKER_IMAGE_NAME):$(DOCKER_BUILD_TAG_PREFIX)-arm64 \
     	$(DOCKER_IMAGE_NAME):$(DOCKER_BUILD_TAG_PREFIX)-arm \
     	$(DOCKER_IMAGE_NAME):$(DOCKER_BUILD_TAG_PREFIX)-amd64)
-	@if [ "$(DOCKER_PUBLISH)"  == "true" ]; then \
```

Comment:
I don't have much experience with bash, but can we use this snippet here?
```
if [[ $(DOCKER_PUBLISH) -eq "true" ]]
```

Reply:
eq is numeric comparison operator, see https://tldp.org/LDP/abs/html/comparison-ops.html

also you are missing quotes around the variable, so empty value won't trigger a syntax error